final_task/rss_parser/requirements.txt

```diff
@@ -0,0 +1,8 @@
+logging
```

Comment:
logging, sys, argparse, json, itertools and re - are parts of stdlibs. You dont need to mention them in requirements file. Hero should go only external libraries, that should be installed via pip

Reply:
Fixed.